ccd-definition/ComplexTypes.json

```diff
@@ -242,5 +242,48 @@
     "HintText": "This should be the type of location you served to, not and address.",
     "SecurityClassification": "Public",
     "FieldShowCondition": "location = \"OTHER\""
+  },
+ {
+    "ID": "CaseDocument",
+    "ListElementCode": "documentName",
+    "FieldType": "Text",
+    "ElementLabel": "Name",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseDocument",
+    "ListElementCode": "documentType",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "DocumentType",
+    "ElementLabel": "Type",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseDocument",
+    "ListElementCode": "documentLink",
+    "FieldType": "Document",
+    "ElementLabel": "Document URL",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseDocument",
+    "ListElementCode": "size",
+    "FieldType": "Number",
+    "ElementLabel": "Document size",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseDocument",
+    "ListElementCode": "createdDatetime",
+    "FieldType": "DateTime",
+    "ElementLabel": "Uploaded on",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "CaseDocument",
+    "ListElementCode": "createdBy",
+    "FieldType": "Text",
+    "ElementLabel": "Uploaded by",
+    "SecurityClassification": "Public"
```

Comment:
@dharmendrak how does this differ from the CCD document type? As this looks like it would be a commonly used complex type, can we discuss this?